lib/bindings/langs/flutter/justfile

```diff
@@ -87,6 +87,10 @@ init *args:
 init-sdk:
 	brew install protobuf
 
+# Update version number on podspec files & CMake scripts
+version:
```

Comment:
Should this be run during the publish build? I imagine the pubspec version being updated but `just version` not run. I don't see the need to keep commiting the podspec.production with version changes

Reply:
> Should this be run during the publish build? I imagine the pubspec version being updated but `just version` not run. I don't see the need to keep commiting the podspec.production with version changes

Yes, I was preparing a separate PR for these changes.

I am also adding a step to main CI to make sure version on podspec files & CMake scripts were updated using this recipe.

A little note: This script doesn't cover the case where version on Flutter plugin's `pubspec.yaml`  wasn't updated properly, which is used as the source version number on this script.

Reply:
@dangeross Done.
- Update versions on podspec & cmake scripts on publish d35ac7e
- Check if versions on podspec & cmake scripts were updated properly on CI 19512d1